lms/djangoapps/teams/serializers.py

```diff
@@ -112,7 +112,8 @@ class MembershipSerializer(serializers.ModelSerializer):
             view_name='teams_detail',
             read_only=True,
         ),
-        expanded_serializer=CourseTeamSerializer(read_only=True)
+        expanded_serializer=CourseTeamSerializer(read_only=True),
+        catch_expand_fields={'user'}
```

Comment:
@dan-f my reading of the code is that this is the list of fields to not expand in the child serializer. It is saying that when we expand teams, we never want to expand users even if the API requested that users be expanded.

I see that @peter-fogg renamed it the way I suggested. Let me know if I'm misunderstanding your point.